_posts/2016-12-04-contributing-blog-post-to-pulpproject.org.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Contributing a Blog Post to pulpproject.org
+author: Dennis Kliban
+---
+Quickstart Guide
+----------------
+
+1. Fork the pulpproject.org repository on GitHub.
+   [https://github.com/pulp/pulpproject.org/](https://github.com/pulp/pulpproject.org/)
+
+1. Clone your fork locally and checkout the `gh-pages` branch.
+
+1. Create a new file in the `_posts` directory. The name for the file should be in the following
+   format: `YYYY-MM-DD-title-of-post.md` e.g.
+   `2016-11-30-contributing-blog-post-to-pulpproject.org.md`
+
+1. Write a blog post in the new file.
+
+1. Commit your changes to the `gh-pages` branch.
+
+1. Push your `gh-pages` branch to GitHub.
+
+1. Preview the blog at `https://<username>.github.io/pulpproject.org/blog/`. You can also build the
+   site locally and view your blog post. More information on building the site can be found in the
+   [README](https://github.com/pulp/pulpproject.org/blob/gh-pages/README.md).
+
+1. Create a Pull Request against the `gh-pages` branch of [pulproject.org
+   repository](https://github.com/pulp/pulpproject.org/).
+
+Writing the Blog Post
+---------------------
+
+Pulpproject.org is a [GitHub Pages](https://pages.github.com/) site. Each page is created using
+[GitHub flavored Markdown](https://help.github.com/articles/basic-writing-and-formatting-syntax/).
+Each blog post begins with the [YAML Front Matter](http://jekyllrb.com/docs/frontmatter/). Front
+Matter begins and ends with `---`. In between the dashes, the author specifies an `author` and a
```

Comment:
s/Front Matter begins and ends with ---. In between the dashes, the author specifies an author and a title/Front Matter species metadata such as the `author` and `title`.